kiuwan/operations.py

```diff
@@ -89,6 +89,65 @@ def check_payload(payload):
     return updated_payload
 
 
+def convert_datetime_in_format(date_time):
+    date_time = date_time.split(".")
+    return date_time[0] + 'Z'
+
+
+def get_application_list(config, params):
+    sort_dict = {
+        'Application Name': 'applicationName',
+        'Analysis Date': 'analysisDate'
+    }
+    kw = Kiuwan(config)
+    endpoint = 'applications/list'
+    initDateAnalysis = params.get('initDateAnalysis')
+    if initDateAnalysis:
+        initDateAnalysis = convert_datetime_in_format(initDateAnalysis)
+    endDateAnalysis = params.get('endDateAnalysis')
+    if endDateAnalysis:
+        endDateAnalysis = convert_datetime_in_format(endDateAnalysis)
+    orderBy = params.get('orderBy')
+    if orderBy:
+        orderBy = sort_dict.get(orderBy)
+    payload = {
+        'applicationName': params.get('app_name'),
+        'activityInfo': params.get('activityInfo'),
+        'initDateAnalysis': initDateAnalysis,
```

Comment:
You can create a common function to build required params from available params and use same in diff functions/actions.

Reply:
Many of the keys name are different for all operations, that's the reason we cannot use same function for payload.